src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -240,6 +266,24 @@ protected static VCFHeader generateVcfHeader(Set<String> sampleNames,
         return header;
     }
 
+    /**
+     * Enforce that if cost information is being recorded to the cost-observability-tablename then *all* recorded
+     * parameters are set
+     */
+    @Override
+    protected String[] customCommandLineValidation() {
+        final List<String> errors = new ArrayList<>();
+        if (projectID != null || datasetID != null || costObservabilityTableName != null || callSetIdentifier != null || wdlStep != null || wdlCall != null || shardIdentifier != null) {
+            if (projectID == null || datasetID == null || costObservabilityTableName == null || callSetIdentifier == null || wdlStep == null || wdlCall == null || shardIdentifier == null) {
+                errors.add("Parameters 'project-id', 'dataset-id', `cost-observability-tablename`, `call-set-identifier`, 'wdl-step', 'wdl-call', and 'shardIdentifier' must either ALL be set or ALL NOT BE set");
```

Comment:
mix of single quotes and backticks around params

Reply:
Thanks!